[Console] Added documentation about command profiling (#19265)
symfony/symfony-docs: console.rst

When a command is running, many events are dispatched, one of them allows to
react to signals, read more in :doc:`this section </components/console/events>`.

> Profiling a command

Reviewer: ```suggestion
Profiling a Command
```

Reviewer 2: Or maybe

Profiling Commands

> -------------------

Reviewer 2: @alamirault the error message from doctor-rst is still wrong (cutted) could you please have a look?

Reviewer: I'll add it to my todo list :smiley:

When debug mode is enabled, you can run a command with the ``--profile`` option. Symfony will then collect data
about the command's execution. When the execution is over, the profile is accessible through the :doc:`Symfony Profiler </profiler>`.

.. tip::

>         If you run the command in verbose mode (-v), Symfony will display in the output a clickable link to the command profile (if your terminal supports links).
>         If you run it in debug verbosity (-vvv) you'll also see the time and memory consumed by the command.

Reviewer 3: ```suggestion
        If you run the command in verbose mode (``-v``), Symfony will display in the output a clickable link to the command profile (if your terminal supports links).
        If you run it in debug verbosity (``-vvv``) you'll also see the time and memory consumed by the command.
```

        .. code-block:: terminal

            $ php bin/console --profile -v app:my-command
            ...

            [OK] Command successful !

            See profile f4ef63 # <- this is a clickable link if your terminal supports it

```

.. versionadded:: 6.4

>     The ``--profile`` option was introduced in Symfony 6.4

Reviewer 3: ```suggestion
    The ``--profile`` option was introduced in Symfony 6.4.
```

When a command is running, many events are dispatched, one of them allows to
react to signals, read more in :doc:`this section </components/console/events>`.

Profiling Commands
------------------

> When debug mode is enabled, you can run a command with the ``--profile`` option.

Reviewer 4: ```suggestion
When debug mode and the profiler are enabled, you can run a command with the ``--profile`` option.
```

> Symfony will then collect data about the command's execution. When the execution

Reviewer 4: ```suggestion
Symfony will then collect data about the command execution. When the execution
```

> is over, the profile is accessible through the :doc:`Symfony Profiler </profiler>`.

Reviewer 4: ```suggestion
is over, the profile is accessible through the web pages of the :doc:`Symfony Profiler </profiler>`.
```
